### ISSUE TITLE
Getting epoch number in training

### ISSUE THREAD

**Issue by fzehracetin** [JiahuiYu/generative_inpainting#437]
Hi @JiahuiYu,
First of all, thank you for your great work. I want to make a change in mask creation mechanism based on an epoch which currently training.  So I just want to ask you how to get epoch number in inpaint_ops.py?

**Comment by JiahuiYu** [comment]
@fzehracetin You can use tensorflow global_step.

**Comment by fzehracetin** [comment]
Hi @JiahuiYu,

In train.py I created the global_step_tensor like this:
`global_step_tensor = tf.Variable(0, trainable=False, name='global_step')`
`d_optimizer = tf.train.AdamOptimizer(lr, beta1=0.5, beta2=0.999)`
`d_optimizer.minimize(losses['d_loss'], global_step=global_step_tensor)`
` g_optimizer = d_optimizer`

But in inpaint_ops.py
`global_step = tf.train.get_global_step()`
It returns None.

I am not experienced on Tensorflow.  Your help is very valuable for me. Thank you for your help.

**Comment by fzehracetin** [comment]
Hey @JiahuiYu,
I'm sorry to bother you again. I can't pass the global_step variable to neuralgym.train.MultiGPUTrainer. I saw that neuralgym.train.trainer has init_primary_trainer() method and the global_step variable can be initialized with it. But how can I do that with MultiGPUTrainer?
